package.json

```diff
@@ -18,7 +19,9 @@
   },
   "jest": {
     "scriptPreprocessor": "preprocessor.js",
-    "unmockedModulePathPatterns": ["node_modules/react"]
```

Comment:
tried running it and kept getting this error:

![image](https://cloud.githubusercontent.com/assets/437318/12357936/cf8042de-bb78-11e5-956e-d88512a986bd.png)